src/bindings/python/flux/Makefile.am

```diff
@@ -34,10 +34,6 @@ nobase_fluxpy_PYTHON = \
 	job/validator/plugins/schema.py \
 	job/validator/plugins/feasibility.py \
 	job/validator/plugins/require-instance.py \
-	job/frobnicator/__init__.py \
```

Comment:
Hmm this is the piece I don't understand - I must have messed something up with creating this second version to delete these lines.